app/assets/javascripts/app/views/notifications_view.js

```diff
@@ -80,6 +78,8 @@ app.views.Notifications = Backbone.View.extend({
       }
     });
 
+    header_badge.text(this.getAllUnread().length());
+
```

Comment:
I've just noticed this bug : when you toggle the dropdown, the badge count is not necessarily updated if you didn't mark an item as read or unread. This force the count to be updated each time you toggle.